src/core_modules/capture-core/components/Relationships/Relationships.component.js

```diff
@@ -144,13 +149,14 @@ class RelationshipsPlain extends React.Component<Props> {
                         enabled={!canDelete}
                         wrapperClassName={classes.tooltip}
                     >
-                        <IconButton
+                        <Button
                             data-test="delete-relationship-button"
                             onClick={() => { onRemoveRelationship(relationship.clientId); }}
                             disabled={!canDelete}
-                        >
-                            <IconCross24 />
-                        </IconButton>
+                            icon={<IconCross24 />}
```

Comment:
To match the design of the trackedEntity relationship widget's delete functionality, can we use [IconDelete16](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetsRelationship/common/RelationshipsWidget/DeleteRelationship/DeleteRelationship.js#L46) instead of IconCross?